### PR TITLE
Pug login form

### PR DESCRIPTION
**Pull request by jon-wehner**

### CODE REVIEW

**Review by ycherradi — APPROVED** [review]

File: routes/index.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
 var router = express.Router();
 
 /* GET home page. */
-router.get('/', function(req, res, next) {
-  res.render('index', { title: 'a/A Express Skeleton Home' });
+router.get("/", function (req, res, next) {
+  res.render("index", { title: "a/A Express Skeleton Home" });
+});
+
+router.get("/rob", (req, res, next) => {
```
Comment (WJVincent) [inline routes/index.js]: We should remove the test route

File: views/login.pug [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                        span.blue b
+                        | lue
+                        span.blue p
+                        | rint
+                    h2 Plan Your Success
+            .registration-form
+                p.login-text Sign Up
+                    |  / 
+                    span.inactive-selection Login
+                form.form(action="users/signup" method="POST")
```
Comment (WJVincent) [inline views/login.pug:25]: We will need to add a _csurf field to the form

**Review by WJVincent — CHANGES_REQUESTED** [review]
See commented changes. We should also delete the raw html files now that they have been converted to pug template files.

**Review by WJVincent — APPROVED** [review]
Looks good!

**Pull request merged**